multiworld/core/image_env.py

```diff
@@ -72,7 +71,10 @@ def __init__(
         # Flattened past image queue
         # init camera
         if init_camera is not None:
-            sim = self._wrapped_env.initialize_camera(init_camera)
+            if hasattr(self._wrapped_env, 'initialized_camera'):
```

Comment:
Hmmm, I feel like this should be the responsibility of the caller to know that the wrapped env doesn't have an initialize_camera function. Otherwise can this silently skip this statement. What do you think?

Reply:
So note that the `hasattr` checks for a different variable `initialized_camera`, not the function. 

The reason for this change is: when using texture randomization it errors unless the camera is set up before the first reset. So the camera needs to be initialized inside the environment __init__, and passed to image_env later on. There might be a way to do this though...

Reply:
I mean, why can't the caller do:
```
env = ...
if hasattr(self._wrapped_env, 'initialized_camera'):
  init_camera = None
env = ImageEnv(env, init_camera=init_camera
```
though ideally we wouldn't use `hasattr`, and it'd be more like:
```
env = ...
if isinstance(env, TexturedEnv):
  init_camera = None
env = ImageEnv(env, init_camera=init_camera
```

Reply:
Moved this logic into the environment and reverted this change to ImageEnv